crates/red_knot_python_semantic/resources/mdtest/scopes/moduletype_attrs.md

```diff
@@ -151,6 +151,7 @@ typeshed = "/typeshed"
 `/typeshed/stdlib/builtins.pyi`:
 
 ```pyi
+class object: ...
```

Comment:
This was required to avoid a salsa cycle. See related discussion https://github.com/astral-sh/ruff/pull/16428#discussion_r1975823169